### PR TITLE
2 Receiver API

### PR DESCRIPTION
**Pull request by topocount**

### CODE REVIEW

File: packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import test from 'ava'
+import getForwardedActions from './get-forwarded-actions'
+import { from } from 'rxjs'
+
+test('should receive and filter the forwardedActions registry', async (t) =>{
+  t.plan(2)
+
+  const forwardedActionsObservable = from([
```
Comment (Quazia) [inline packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.test.js:8]: I would make some of these different, and test against a unique one. In theory, the observable should never have duplicates in it like this right? Also, ensuring access to more than just the first value would probably be good. As it is it's just passing as soon as the first value is read in so having multiples doesn't really make sense.

**Review by Quazia — CHANGES_REQUESTED** [review]
LGTM other than a minor change to the tests

File: packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const getProxyActions = actionsArray => (
+    actionsArray
+    .filter(action => (action.target === proxy.address))
+  )
+
+  return wrapper.forwardedActions.pipe(
+    // transform the observable into an event-like object
+    // that only contains actions targeting the proxy
+    map(actions => ({
+      event: 'ForwardedActions',
```
Comment (Quazia) [inline packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.js:15]: I think the structure we want might be `event: { eventName: 'ForwardedActions', returnValues: getProxyActions(actions) }`

Comment (topocount) [reply packages/aragon-wrapper/src/rpc/handlers/get-forwarded-actions.js:15]: we alias `event` to be `eventName` within our script.js files: https://github.com/AutarkLabs/planning-suite/blob/6927c8fb9d250218c096b720fc52e509a118ad2c/apps/rewards/app/store/events.js#L7

**Review by Quazia — APPROVED** [review]
LGTM

**Pull request merged**